src/api/models/groupChat.js

```diff
@@ -19,6 +24,5 @@ var groupMessageSchema = new mongoose.Schema(
     },
   },
 );
-groupMessageSchema.index({ name: 1 });
-
-export default mongoose.model('GroupMessage', groupMessageSchema);
+groupChatSchema.index({ name: 1 });
```

Comment:
We don't need to create index with name field since we don't query follow group chat name